kwok/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Start from a base image
+FROM registry.access.redhat.com/ubi8:latest
+
+# Update packages
+RUN yum update -y && yum clean all
+
+RUN yum install -y bash
```

Comment:
No way you need this, bash is built-in to the image

Also because this is a sperate RUN step - you leave the yum cache in the image layer.

Reply:
removed